docs/a2a/quickstart-exposing.md

```diff
@@ -63,6 +63,32 @@ a2a_app = to_a2a(root_agent, port=8001)
 
 The `to_a2a()` function will even auto-generate an agent card in-memory behind-the-scenes by [extracting skills, capabilities, and metadata from the ADK agent](https://github.com/google/adk-python/blob/main/src/google/adk/a2a/utils/agent_card_builder.py), so that the well-known agent card is made available when the agent endpoint is served using `uvicorn`.
 
+You can also provide your own agent card by using the `agent_card` parameter. The value can be an `AgentCard` object or a path to an agent card JSON file.
+
+**Example with an `AgentCard` object:**
+```python
+from google.adk.a2a.utils.agent_to_a2a import to_a2a
+from a2a.types import AgentCard
+
+# Make your agent A2A-compatible with a custom agent card
```

Comment:
statement is misleading / confusing. This step does not "Make your agent A2A-compatible", it just adds the agent card data.